components/private/ThumbTrack/src/MDCThumbTrack.m

```diff
@@ -822,14 +842,21 @@ - (void)updateViewsMainIsAnimated:(__unused BOOL)animated
     if ([_delegate respondsToSelector:@selector(thumbTrack:stringForValue:)]) {
       _valueLabel.text = [_delegate thumbTrack:self stringForValue:_value];
       if (CGRectGetWidth(_valueLabel.frame) > 1) {
+        CGFloat scale = UIScreen.mainScreen.scale;
```

Comment:
Nit: Please use the view's current screen's scale (if available).  Something like this:

```objc
CGFloat scale = self.window.screen.scale > 0 ?: UIScreen.mainScreen.scale;
```

Reply:
done